src/api/spec/controllers/webui/request_controller_spec.rb

```diff
@@ -76,4 +76,44 @@
       end
     end
   end
+
+  describe "POST #change_devel_request" do
+    context "with valid parameters" do
+      before do
+        login(submitter)
+        post :change_devel_request, project: target_project.name, package: target_package.name,
+          devel_project: source_project.name, devel_package: source_package.name, description: "change it!"
+        @bs_request = BsRequest.where(description: "change it!", creator: submitter.login, state: "new").first
+      end
+
+      it { expect(response).to redirect_to(request_show_path(number: @bs_request)) }
+      it { expect(flash[:success]).to be nil }
+
+      it "creates a delete request with the correct data" do
+        expect(@bs_request).not_to be nil
+        expect(@bs_request.description).to eq("change it!")
```

Comment:
Why not splitting this up in single ``expects``?
And also I'm wondering if a simple

``
expect { post }.to change{ BsRequest.count }.by(1)
``

wouldn't be sufficient?